test/prow-cluster-operation/main.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"log"
+
+	"knative.dev/test-infra/test/prow-cluster-operation/actions"
+	"knative.dev/test-infra/test/prow-cluster-operation/options"
+)
+
+func main() {
+	create := flag.Bool("create", false, "Create cluster")
+	delete := flag.Bool("delete", false, "Delete cluster")
+	get := flag.Bool("get", false, "Get cluster")
+	o := options.NewRequestWrapper()
+	o.AddOptions()
```

Comment:
Can we add the options in the function `options.NewRequestWrapper()`, and remove this `AddOptions` functions?  It looks unnecessary and is adding some confusion.

Reply:
done